src/quo2/components/list_items/user_list/style.cljs

```diff
@@ -0,0 +1,40 @@
+(ns quo2.components.list-items.user-list.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn user-list-wrapper [can-be-invited? background-type]
+  (let [height (if can-be-invited? 56 80)]
+    {:width              359
+     :border-radius      12
+     :padding-vertical   8
+     :padding-horizontal 12
+     :height             height
+     :background-color   (case background-type
+                           :5  colors/primary-50-opa-5
+                           :10 colors/primary-50-opa-10
+                           :transparent :transparent)
+     :flex-direction     :column
+     :justify-content    :center}))
+
+(def ml-8 {:margin-left 8})
```

Comment:
There's nothing wrong with this per se, but giving a name for a margin left seems excessive. You actually reminded me of Tailwind CSS or Bootstrap because of utility class names like `ml-8`, which I'm actually quite fond of :)